etp-backend/src/main/clj/solita/etp/service/energiatodistus.clj

```diff
@@ -165,17 +168,20 @@
   (map (comp flat->tree db/kebab-case-keys)
        (energiatodistus-db/select-sisaiset-kuormat db {:versio versio})))
 
-(def db-row->energiatodistus
-  (comp coerce-energiatodistus
+(defn db-row->energiatodistus-f [schema]
```

Comment:
ehkä selkeämpi nimi voisi olla db-row->energiatodistus-schema - kaikkihan nämä lähtökohtaisesti on funktioita

Reply:
Joo, mutta tämä palauttaa funktion, eikä energiatodistusta. Ei ole minustakaan paras mahdollinen nimi tosin, mutta jotenkin pitäisi tuoda se ilmi että tällä rakennetaan se varsinainen funktio, jota käytetään.

Reply:
tarkoittaako -f siis paluuarvon tyyppiä?

Reply:
Joo. Oisko joku parempi ehdotus? Mietin myös *-notaatiota, mutta ei sekään ole tässä hyvä sillä juuri tuota -f -versiota kutsutaan useimmiten muista nimiavaruuksista.

Reply:
entäs: schema->db-row->energiatodistus ?

Reply:
Tuo vois olla hyvä, vaikka en tiedä kuinka Clojuremaista tuo on. Muokkaan ja mergeen.

Reply:
joo meneehän toi -> vähän päällekkäin thread makrojen kanssa, mutta tuskin noita voi niihin sekoittaa